pkg/oc/bootstrap/docker/openshift/helper.go

```diff
@@ -251,6 +251,9 @@ func (h *Helper) OtherIPs(excludeIP string) ([]string, error) {
 	candidates := strings.Split(result, " ")
 	resultIPs := []string{}
 	for _, ip := range candidates {
+		if len(strings.TrimSpace(ip)) == 0 {
```

Comment:
@deads2k this is funny... we always passed '\n' as first item here for some reason...

Reply:
that's inspiring